components/consistencybox.tsx

```diff
@@ -81,38 +82,15 @@ const ConsistencyBox: React.FC<{}> = () => {
           <div style={{ marginBottom: 10, fontWeight: 'bold' }}>{title}</div>
           <div>{content}</div>
         </CardContent>
-        <CardContent>Bruk konsistenssjekk:</CardContent>
-        <CardActions>
-          <Switch
-            checked={checked}
-            onChange={handleCheck}
-            inputProps={{ 'aria-label': 'controlled' }}
-          />
-        </CardActions>
-        <CardContent> Andel besvarelser som skal revurderes:</CardContent>
-        <CardActions disableSpacing>
-          {checked ? (
-            <Slider
-              sx={{ marginLeft: '1.2rem', marginRight: '1rem' }}
-              aria-label="Custom marks"
-              defaultValue={10}
-              getAriaValueText={valuetext}
-              step={10}
-              valueLabelDisplay="auto"
-              marks={marks}
-            />
-          ) : (
-            <Slider
-              sx={{ marginLeft: '1.2rem', marginRight: '1rem' }}
-              disabled
-              aria-label="Custom marks"
-              defaultValue={10}
-              getAriaValueText={valuetext}
-              step={10}
-              valueLabelDisplay="auto"
-              marks={marks}
+        <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+          <CardContent>Bruk konsistenssjekk:</CardContent>
+          <CardActions>
+            <Switch
+              checked={checked}
+              onChange={handleCheck}
+              inputProps={{ 'aria-label': 'controlled' }}
             />
-          )}
+          </CardActions>
           <ExpandMore
             sx={{ marginLeft: '3rem' }}
```

Comment:
```suggestion
            sx={{ marginLeft: '3rem' , marginRight: '0.5rem' }}
```

Reply:
To make it more aligned with the other expand components

Reply:
genius:)